WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -591,6 +601,13 @@ class PostCoordinator: NSObject {
         }
     }
 
+    private func shouldScheduleRetry(for error: Error) -> Bool {
+        if error is PostRepository.PostSaveError {
```

Comment:
I would suggest checking for a specific case and making it so that you need to opt-out of retries if you add new error cases.

For `.deleted` (aka not found) this branch is unreachable.

Reply:
Good point - fixed! https://github.com/wordpress-mobile/WordPress-iOS/pull/22992/commits/02cea1b147580af4e1fc3d30a432703a2a976068